src/api.ts

```diff
@@ -26,3 +26,13 @@
 export const getTodos = () => get<Todo[]>('/todos');
 
 export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+
+export const getComplited = () => {
+  return getTodos().then(todos => todos.filter(todo => todo.completed));
+};
+
+export const getActive = () => {
```

Comment:
Consider renaming the function to follow the naming conventions for methods. A more descriptive name like 'fetchActiveTodos' would be clearer.